MAUI/ImageEditor/zooming.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+title: Zoom image using .NET MAUI Image Editor Control | Syncfusion
+description: Learn here all about how to zoom image in Syncfusion .NET MAUI Image Editor(SfImageEditor) control and more.
+platform: .NET MAUI
+control: SfImageEditor
+documentation: ug
+---
+
+# .NET MAUI Image Editor (SfImageEditor) Image Zooming
+
+The image editor control in .NET MAUI provides support for zooming and panning actions on an image.
+
+## Allow image zooming
+
+To enable or disable the zooming functionality, set the value of the [`AllowZoom`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_AllowZoom) property to true or false. By default, the [`AllowZoom`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_AllowZoom) value is set to `true`.
+
+{% tabs %}
+
+{% highlight XAML %}
+  <ContentPage 
+            . . .
+            <imageEditor:SfImageEditor Source="image.jpeg" AllowZoom="False"/>
+
+   </ContentPage>
+     
+{% endhighlight %}
+
+{% highlight C# %}
+   
+     SfImageEditor imageEditor = new SfImageEditor();
+     imageEditor.Source = "image.jpeg";
+     imageEditor.AllowZoom = false;
+     this.content = imageEditor;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Change image zoom level
+
+Programmatically, you can zoom the loaded image without any interaction by setting a value to [`ZoomLevel`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_ZoomLevel) property. 
```

Comment:
Programmatically, you can zoom the loaded image without any interaction by setting a value to the [`ZoomLevel`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_ZoomLevel) property.